fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -337,7 +338,7 @@ export class FlightPlanManager {
      * @param callback A callback to call when the operation has completed.
      */
     public async setOrigin(icao: string, callback = () => { }): Promise<void> {
-        const sameAirport = this.getOrigin()?.ident === icao;
+        const sameAirport = ICAO.getIdent(icao) === icao;
```

Comment:
Maybe better to use `WayPoint.formatIdentFromIcao` to match all of our other code and avoiding mixing old+new WT SDK, and it handles some other cases that ICAO.getIdent doesn't.

Reply:
Wait... this code really doesn't make sense.